lib/Dialect/FIRRTL/Transforms/BlackBoxReader.cpp

```diff
@@ -53,59 +54,36 @@ using namespace firrtl;
 
 namespace {
 
-/// This is used to indicate the directory priority.  Multiple external modules
-/// with the same "defname" may have different output filenames.  This is used
-/// to choose the best filename.
-enum class Priority { TargetDir = 0, Verification, Explicit, TestBench, Unset };
-
 /// Data extracted from BlackBoxInlineAnno or BlackBoxPathAnno.
 struct AnnotationInfo {
   /// The name of the file that should be created for this BlackBox.
   StringAttr name;
-  /// The output directory where this annotation should be written.
-  StringAttr outputFile;
+  /// The outputdirectory information for this extmodule.
```

Comment:
```suggestion
  /// The output directory information for this extmodule.
```

Reply:
Fixed in: https://github.com/llvm/circt/pull/7541/commits/836291f8d0bb35068f1719634578a26b962a9902